.gitlab-ci.yml

```diff
@@ -332,7 +332,7 @@ variables:
     variables:
       AGENT_REPOSITORY: agent
       DSD_REPOSITORY: dogstatsd
-      IMG_REGISTRIES: public
+      IMG_REGISTRIES: internal-aws-ddbuild,public
```

Comment:
Note that this will publish to `<internal>/dogstatsd` or `<internal>/agent`, which currently do not exist.
Also I think we don't want to mix up internal release and external releases.

You may need a dedicated job for that, with dedicated repos (or use the `-release` repo but with specific tags, like `public-`).